DbTables/inc/TrkDelayPreamp.hh

```diff
@@ -0,0 +1,63 @@
+#ifndef DbTables_TrkDelayPreamp_hh
+#define DbTables_TrkDelayPreamp_hh
+
+
+#include <string>
+#include <iomanip>
+#include <sstream>
+#include <map>
+#include "DbTables/inc/DbTable.hh"
+
+namespace mu2e {
+
+  class TrkDelayPreamp : public DbTable {
+  public:
+
+    typedef std::shared_ptr<TrkDelayPreamp> ptr_t;
+    typedef std::shared_ptr<const TrkDelayPreamp> cptr_t;
+
+    class Row {
+    public:
+      Row(int index, float delay_hv, float delay_cal):_index(index),
+        _delay_hv(delay_hv), _delay_cal(delay_cal) {}
+      int  index() const { return _index;}
+      float delayHv() const {return _delay_hv;}
+      float delayCal() const {return _delay_cal;}
+    private:
+      int _index;
+      float _delay_hv;
+      float _delay_cal;
+    };
+
+    constexpr static const char* cxname = "TrkDelayPreamp";
+
+    TrkDelayPreamp():DbTable(cxname,"trk.delaypreamp",
+	 "index,delay_hv,delay_cal") {}
+    const Row& rowAt(const std::size_t index) const { return _rows.at(index);}
+    std::vector<Row> const& rows() const {return _rows;}
+    std::size_t nrow() const override { return _rows.size(); };
+    virtual std::size_t nrowFix() const override { return 96; }; 
```

Comment:
This being 96 instead of 20k seems to contradict the title of the PR.  I think we can take it from StrawId too. I think the ambiguity suggests index should be called, effectively, sequentialStrawIndex.  Long ago, we agreed on "RStraw" to mean 0-95, "reduced straw index".  Maybe index can be simply straw or sstraw for sequential straw index.  Similarly, I'm not sure if "preamp" implies the meaning of index, or a meaning of the content.  Another way to make the indexing clear is to call the table TrkStrawDelayPreamp, or something that indicates a row relates to a straw.   I'd like to have conventions, so for example, *Straw* tables are all 20k rows, but maybe it is too much work.

Reply:
This is actually something different now, it represents the part of the delay that is fixed the same in all panels due to trace lengths, so it is actually only 96 rows. The 20k part is turning on using TrkPreampStraw instead of TrkPreampRStraw for the rest of the parameters. We could call it TrkDelayRStraw if we want that to be the convention for 96 row tables

Reply:
Ah, OK, I see you had the straw table there, but had not used it until now.  If the index of the new table is the same as the previous "RStraw" tables, then I'd appreciate if the new table were also "RStraw".  I really think this sort of consistency helps with understanding things at a glance.